### DISCUSSION TITLE
11 proof but 0 Chia wins

### DISCUSSION THREAD

**Discussion by BianKo666niglio** [Chia-Network/chia-blockchain#7498] [Support]
Hi everybody.

Is normal to have 11 proof founded but 0 Chia?

I check my harvester and 

Avarage lookup time 0,18s
Worst lookup time 3,2s
Best lookup time 0,005s

181 plots (22 NFTplot and 159 OGplot)

Thanks in advance

**Comment by d1Mm** [comment]
finding proof on the NFTplot gives you pool points, not coins, at least until you switch NFT to self pooling

**Reply by turbotas** [reply]
Those average, worst and best lookup times are pretty good.  With 159 OGplot, it will be many months before you can statistically expect to win so those 11 proofs won't be against those plots. If you check with your pool operator, there should be stats available about how many challenges they have given you for the 22 NFT plots and maybe those numbers will tie up.  As d1Mm said, those 11 proofs won't be against your 159OGplots.
